.circleci/config.yml

```diff
@@ -8,23 +8,28 @@ update_submodule: &update_submodule
 
 setup_base_docker: &setup_base_docker
   name: Setup Base Docker Image
-  command: .circleci/setup_ci_environment.sh
+  command: |
+    .circleci/setup_ci_environment.sh
 
 launch_docker_and_build: &launch_docker_and_build
   name: Launch Docker Container and Build
   no_output_timeout: "1h"
   command: |
     set -e
     eval $(aws ecr get-login --region us-east-1 --no-include-email)
-    time docker pull ${DOCKER_IMAGE} >/dev/null
+    time docker pull ${GCR_DOCKER_IMAGE} >/dev/null
+    time docker tag ${GCR_DOCKER_IMAGE} ${ECR_DOCKER_IMAGE} >/dev/null
+    time docker push ${ECR_DOCKER_IMAGE} >/dev/null
```

Comment:
This is to stage our base image for use in the upstream. Right now, we are using their base image in the upstream test -- but, in case we also want the upstream workflow to use our build image, we need to have them in the ECR. This is not expensive, and only pushes it if the image layers are not present in the repo.

Reply:
Can you add this comment to the yml file as comment?

Reply:
nit, replace `their` to `pytorch` and `our` to `torch/xla`